portfolio/src/main/webapp/index.html

```diff
@@ -40,6 +40,12 @@ <h2>About Me </h2>
   <p>Hi There, I am Prashant Neupane all the way from Nepal. Nepal is a small
       landlocked country in the Himalayas. I go to Fisk University in Nashville, TN.
       I am a Computer Science Major and rising Junior.
+  <button onclick="showComments()">Show Comments!</button>
+  <div id="greeting-container"></div>
```

Comment:
Is the `greeting-container` element still used or can it be removed?

Reply:
It is not used anymore. Removing it
Changes made.